src/editor/tests.py

```diff
@@ -242,125 +242,247 @@ def test_editor_review_round_cancel(self):
-                                        kwargs={'submission_id': self.book.id,
-                                                'review_type': 'internal'}),
-                                {'file': 2})
+        resp = self.client.post(
+            reverse(
+                'add_review_files',
+                kwargs={
+                    'submission_id': self.book.id,
+                    'review_type': 'internal'
+                }
+            ),
+            {'file': 2}
+        )
         self.assertEqual(resp.status_code, 302)
-        self.assertEqual(resp['Location'],
-                         "http://testing/editor/submission/1/review/round/2/")
+        self.assertEqual(
+            resp['Location'],
+            "http://testing/editor/submission/1/review/round/2/"
+        )
 
-        resp = self.client.post(reverse('add_review_files',
-                                        kwargs={'submission_id': self.book.id,
-                                                'review_type': 'external'}),
-                                {'file': 2})
+        resp = self.client.post(
+            reverse(
+                'add_review_files',
+                kwargs={
+                    'submission_id': self.book.id,
+                    'review_type': 'external'
+                }
+            ),
+            {'file': 2}
+        )
         self.assertEqual(resp.status_code, 302)
-        self.assertEqual(resp['Location'],
-                         "http://testing/editor/submission/1/review/round/2/")
+        self.assertEqual(
+            resp['Location'],
+            "http://testing/editor/submission/1/review/round/2/"
+        )
 
-        resp = self.client.post(reverse('delete_review_files',
-                                        kwargs={'submission_id': self.book.id,
-                                                'review_type': 'external',
-                                                'file_id': 2}))
+        resp = self.client.post(
+            reverse(
+                'delete_review_files',
+                kwargs={
+                    'submission_id': self.book.id,
+                    'review_type': 'external',
+                    'file_id': 2
+                }
+            )
+        )
         self.assertEqual(resp.status_code, 302)
-        self.assertEqual(resp['Location'],
-                         "http://testing/editor/submission/1/review/round/2/")
-        resp = self.client.post(reverse('delete_review_files',
-                                        kwargs={'submission_id': self.book.id,
-                                                'review_type': 'internal',
-                                                'file_id': 2}))
+        self.assertEqual(
+            resp['Location'],
+            "http://testing/editor/submission/1/review/round/2/"
+        )
+        resp = self.client.post(
+            reverse(
+                'delete_review_files',
```

Comment:
Going by the name of this URL, I'm assuming it deletes a fie. Should there be a test to see if this is done?

Reply:
Well spotted, there's little point making the request and not checking that it's worked. `delete_review_files` actually removes a file from a ManyToManyField on the Book model. I've added an `assertNotIn` for this.